specs/data_structures.md

```diff
@@ -390,7 +390,7 @@ Reed-Solomon erasure coding is used as the underlying coding scheme. The paramet
 - [`availableDataOriginalSquareSize`](#header) parity pieces (maximum of [`AVAILABLE_DATA_ORIGINAL_SQUARE_MAX`](./consensus.md#constants)) (i.e `availableDataOriginalSquareSize * 2` total pieces), for an erasure efficiency of 50%. In other words, any 50% of the pieces from the `availableDataOriginalSquareSize * 2` total pieces are enough to recover the original data.
 - [`SHARE_SIZE`](./consensus.md#constants) bytes per piece
 
-[Leopard-RS](https://github.com/catid/leopard) is a C library that implements the above scheme with quasilinear runtime.
+Note that [`availableDataOriginalSquareSize`](#header) may vary each block. [Leopard-RS](https://github.com/catid/leopard) is a C library that implements the above scheme with quasilinear runtime.
```

Comment:
Can we clarify further how it exactly it varies and what this depends on? Ideally, this could be expressed as a function? Does it depend on the largest Message (which means more shares)?

Reply:
Fixed in 53e5f35.